src/components/Extensions/Details/DetailCard/RelatedLinks/RelatedLinks.tsx

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Icon } from '@rmwc/icon';
+import { Theme } from '@rmwc/theme';
+import { Typography } from '@rmwc/typography';
+import React from 'react';
+
+import { Extension } from '../../../models';
+import style from './RelatedLinks.module.scss';
+
+export interface RelatedLinksProps {
+  extension: Extension;
+}
+
+export const RelatedLinks: React.FC<RelatedLinksProps> = ({ extension }) => {
+  return (
+    <Theme use="secondary">
+      <ul className={style.links}>
+        <li>
+          <Icon icon="person" />
+
+          <Typography use="body2" theme="secondary">
+            Made by{' '}
+            <a
+              href={extension.authorUrl}
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              {extension.authorName}
+            </a>
```

Comment:
authorUrl is optional; so this link should also be optional - and when the url is not defined, just printing the authorName.

Also to double check - is authorName derived from the ref here or is this using the spec.author.name field? We should prefer the former and fallback to the later